VHbb/interface/VHbbNameSpace.h

```diff
@@ -106,6 +109,108 @@ double SoverSBWeight(double BDT, int channel) {
+
+    //Check to which bin the BDT belongs to
+    const int nBins=15;
+    std::map<int, std::vector<double>> binBoundaries;
+
+    binBoundaries.insert(std::pair<int, std::vector<double> > (1, {-0.8, -0.7964, -0.6686000000000001, -0.5408000000000001, -0.41300000000000003, -0.2852, -0.1574000000000001, -0.02960000000000007, 0.09819999999999995, 0.22599999999999998, 0.3537999999999999, 0.48160000000000003, 0.6093999999999999, 0.7407999999999999, 0.8685999999999998, 1.0} ));  //bins['zeelow']
+
+    binBoundaries.insert(std::pair<int, std::vector<double> > (2, {-0.8, -0.7964, -0.6722, -0.548, -0.42380000000000007, -0.2996000000000001, -0.1754000000000001, -0.05120000000000002, 0.07299999999999995, 0.19719999999999993, 0.3213999999999999, 0.4456, 0.5697999999999999, 0.7011999999999998, 0.8253999999999999, 1.0} ));  //bins['zuulow']
+
+    binBoundaries.insert(std::pair<int, std::vector<double> > (3, {-0.8, -0.7766000000000001, -0.6596000000000001, -0.5426, -0.42560000000000003, -0.30860000000000004, -0.1916000000000001, -0.07460000000000011, 0.04239999999999988, 0.15939999999999988, 0.2764, 0.39339999999999997, 0.5104, 0.6399999999999999, 0.7569999999999999, 1.0} ));  //bins['zeehigh'
+
+    binBoundaries.insert(std::pair<int, std::vector<double> > (4, {-0.8, -0.7964, -0.6776000000000001, -0.5588000000000001, -0.44000000000000006, -0.32120000000000004, -0.20240000000000002, -0.08360000000000012, 0.0351999999999999, 0.15399999999999991, 0.27279999999999993, 0.39159999999999995, 0.5104, 0.6436, 0.7624, 1.0} ));  //bins['zuuhigh'
+
+    int BDTbin = -1;
+    for(int binNumber=0; binNumber<nBins; binNumber++){
+        if(BDT>=binBoundaries[channel][binNumber]){
+            BDTbin++;
+        } else {
+            break;
+        }
+    }
+
+    // if BDT<-0.8, this will break; maybe add if(BDT<binBoundaries[channel][0]) return 0; ?
+
+
+    double SB_Zuu_low[nBins]={0.000044488502184739564, 
+			      0.00029649100871548956, 
+			      0.0009201810958585284, 
+			      0.0016066423379721248, 
+			      0.0028950795731256976, 
+			      0.004134903391541092, 
+			      0.0062762848083387, 
+			      0.009604671157499913, 
+			      0.014138042105016676, 
+			      0.021066949772195805, 
+			      0.034176274685107554, 
+			      0.04951176273189254, 
+			      0.07537928093659098, 
+			      0.11575578259764387, 
+			      0.16743353573335515
+    };
+
+    double SB_Zee_low[nBins]={0.0002093144475949369, 
+			      0.0005954847339052542, 
+			      0.0009210055149576992, 
+			      0.0014175050626033137, 
+			      0.0029770671147323594, 
+			      0.004509042207332993, 
+			      0.007810157102722652, 
+			      0.01202560316981223, 
+			      0.016652303574890805, 
+			      0.02328327194270849, 
+			      0.037752499238423, 
+			      0.06556187955627332, 
+			      0.09162826576126447, 
+			      0.15170491815076576, 
+			      0.31970439154057484
+    };
+
+    double SB_Zuu_high[nBins]={0.0, 
```

Comment:
This is the value Sean has on his webpage: https://swang373.web.cern.ch/swang373/sbweights_validation.html